### PR TITLE
Fix debug menu terminology.

### LINKED ISSUE #9375: Inconsistent naming in the lobby notifications
screen shot 2015-09-19 at 12 22 07

"Developer Cheats" should be changed to match the option name.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26800556-inconsistent-naming-in-the-lobby-notifications?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by pchote**
Fixes #9375 and a polish regression introduced by #8700.

### CODE REVIEW

File: OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs [hunk 1]
```diff
@@ -417,8 +417,8 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 
 						bool.TryParse(s, out server.LobbyInfo.GlobalSettings.AllowCheats);
 						server.SyncLobbyGlobalSettings();
-						server.SendMessage("{0} {1} Developer Cheats."
-							.F(client.Name, server.LobbyInfo.GlobalSettings.AllowCheats ? "allowed" : "disallowed"));
+						server.SendMessage("{0} {1} the Debug Menu."
```
Comment (Mailaender) [inline OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs]: This isn't just enabling the menu. We also have command line developer cheats/options.

Comment (pchote) [inline OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs]: The command line offers a shortcut to access a subset of the developer menu options. "Developer Menu" covers both cases, while "Cheats" does not.

Comment (Mailaender) [inline OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs]: Fair enough. :+1:

**Comment by abcdefg30** [comment]
Seems simple enough. :+1:

**Pull request merged**